rules/windows/registry_event/sysmon_dns_serverlevelplugindll.yml

```diff
@@ -30,13 +30,13 @@ logsource:
     category: registry_event
 detection:
     dnsregmod: 
-        TargetObject: '*\services\DNS\Parameters\ServerLevelPluginDll'
+        TargetObject|endswith: '\services\DNS\Parameters\ServerLevelPluginDll'
     condition: 1 of them
 ---
 logsource:
     category: process_creation
     product: windows
 detection:
     dnsadmin:
-        CommandLine: 'dnscmd.exe /config /serverlevelplugindll *'
-    condition: 1 of them
\ No newline at end of file
+        CommandLine|startswith: 'dnscmd.exe /config /serverlevelplugindll '
```

Comment:
better to split ProcessName (dnscmd.exe) and the commandline

Reply:
Fixed